elevate-cpanel

```diff
@@ -232,43 +232,134 @@ use constant GRUB_EDITENV      => '/usr/bin/grub2-editenv';
 use constant GRUB_ENV_FILE     => '/boot/grub2/grubenv';
 
 use constant DISABLE_MYSQL_YUM_REPOS => qw{
-  Mysql57.repo
-  Mysql80.repo
-
-  MariaDB102.repo
-  MariaDB103.repo
-  MariaDB105.repo
-  MariaDB106.repo
-
-  mysql-community.repo
-
+  mysql-connectors-community
+  mysql-tools-community
+  mysql55-community
+  mysql56-community
+  mysql57-community
+  mysql80-community
+  mysql-tools-preview
+  mysql-cluster-7.5-community
+  mysql-cluster-7.6-community
```

Comment:
maybe we should consider RegExp for some of these entries? which would provide an easier maintenance
example: `mysql-cluster-\d+\.\d+-community`

Reply:
Let's do this in a follow up commit